opentaxii/http.py

```diff
@@ -4,9 +4,27 @@
 from .server import TAXIIServer
 from .utils import configure_logging
 
-config = ServerConfig()
-configure_logging(config.get('logging', {'': 'info'}))
 
-server = TAXIIServer(config)
+# This module is also used as a Gunicorn configuration module, i.e. passed
+# as ``--config python:opentaxii.http``. ``logconfig_dict`` module-level
```

Comment:
could you please update related docs? https://github.com/eclecticiq/OpenTAXII/blob/master/docs/running.rst#production-mode

Reply:
Sure. Fixed in ad9b0905840e.